.github/workflows/build.yml

```diff
@@ -4,10 +4,6 @@ on:
   pull_request:
     branches:
       - main
-    paths:
-      - 'kong/**/*.lua'
-      - 'spec/*.lua'
-
```

Comment:
Why remove this?
By removing the Lua paths, it means that the linting workflow will run even if only markdown files are changed.

Reply:
It wasn't running as it was, just updated it to include only `spec` and `kong`